tests/sqllogictests/suites/mode/standalone/explain/window.test

```diff
@@ -371,5 +368,26 @@ CompoundBlockOperator(Project) × 1 processor
                         DeserializeDataTransform × 1 processor
                           SyncReadParquetDataSource × 1 processor
 
+# row fetch with window function
+query T
+explain pipeline select *, sum(a) over (partition by a order by a desc rows between unbounded preceding and current row) from t where a > 1 order by b limit 3;
+----
+CompoundBlockOperator(Project) × 1 processor
+  TransformRowsFetcher × 1 processor
```

Comment:
Seems it's RowFetcher of b, we want to have a.

Reply:
if add `a` to lazy_cols, it will not get right field when do `EvalScalar` for window function.